db/migrate/20200122110833_add_entry_body_to_timeline_entry_whitehall_imported_entries.rb

```diff
@@ -0,0 +1,5 @@
+class AddEntryBodyToTimelineEntryWhitehallImportedEntries < ActiveRecord::Migration[6.0]
+  def change
+    add_column :timeline_entry_whitehall_imported_entries, :entry_body, :json
```

Comment:
I'm not sure you need to prefix this name with entry since the class is already an entry.

Do you want this to be null too? It can be quite common to have JSON fields as non-null and default to an empty hash. Null might well serve a purpose here so you may want it, but just flagging as it can be confusing if you have to check regularly that item isn't null for usage.

Reply:
Thanks for the comments.  Fixed as requested and made JSON field not null with default.